src/Api/AdminConsole/Controllers/ProviderOrganizationsController.cs

```diff
@@ -87,7 +91,6 @@ public async Task Delete(Guid providerId, Guid id)
             throw new NotFoundException();
         }
 
-        var userId = _userService.GetProperUserId(User);
```

Comment:
The queries to obtain `ProviderOrganization` and `Organization` could be performed here in the controller.